src/main/java/net/sf/jabref/JabRefGUI.java

```diff
@@ -65,6 +67,13 @@ public JabRefGUI(List<ParserResult> loaded, boolean isBlank) {
         this.loaded = loaded;
         this.isBlank = isBlank;
         openWindow();
+        JabRefGUI.checkVersion(false);
+    }
+
+    public static void checkVersion(boolean manualExecution){
```

Comment:
Please move this to VersionWorker (or Version?) and rename to checkForNewVersion.